build.gradle.kts

```diff
@@ -1,16 +1,16 @@
 plugins {
     `java-gradle-plugin`
     `kotlin-dsl`
-    kotlin("jvm") version "1.4.31"
-    id("com.gradle.plugin-publish") version "0.15.0"
+    kotlin("jvm") version "1.7.10"
+    id("com.gradle.plugin-publish") version "1.1.0"
 }
 
 repositories {
     mavenCentral()
 }
 
 group = "com.lovelysystems"
-version = "1.10.0"
+version = "1.10.1"
```

Comment:
the version is updated on master when releasing a new version

Reply:
fair enough!